scripts/configs/networks/avalanche.ts

```diff
@@ -13,3 +13,11 @@ export const avalancheAddresses: NetworkAddresses = {
     AAVE_MERKLE_DISTRIBUTOR: '0xA065d5A299E618CD84a87641d5eEbC7916Fdf32E',
   },
 };
+
+export const avalancheFujiAddresses: NetworkAddresses = {
+  name: 'AvaxFuji',
```

Comment:
```suggestion
  name: 'Fuji',
```
Let's make it match with the name in `pools/avalanche.ts`